tests/ui/src/utilities/s3Helper.ts

```diff
@@ -0,0 +1,28 @@
+
+const bucket = `nhse-uec-cm-ui-test-bucket${process.env.WORKSPACE}`;
```

Comment:
The env is missing from the bucket name 
const bucket = `nhse-uec-cm-testing-bucket-${process.env.ENV}${process.env.WORKSPACE}`;